api/pkg/transformer/server/rest/server.go

```diff
@@ -174,6 +175,13 @@ func (s *HTTPServer) PredictHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func responseCodeFromError(err error) int {
+	if errors.Is(err, mErrors.InvalidInputError) {
+		return http.StatusUnprocessableEntity
```

Comment:
Why do you use `StatusUnprocessableEntity` instead of `BadInput` ?

Reply:
do you mean `StatusBadRequest`? my take on `StatusBadRequest` is when the format of incoming request is not valid in term of syntax or type of payload. I choose `StatusUnprocessableEntity` due to it is correct in term of validity of request but failed to be processed due to some logic in our server. But I think people tends to use 400 today  https://www.keycdn.com/support/422-unprocessable-entity, should we use 400 instead?

Reply:
Yeah, I think it's more understandable if we return 400